dluhc-component-library/src/components/maps/baseMap.tsx

```diff
@@ -59,6 +61,11 @@ const BaseMap = ({
     );
     map.setTarget(ref.current);
     map.addInteraction(new Draw({ source: source, type: "Polygon" }));
```

Comment:
Discussed offline, we're going to move the drawing into it's own component in a future PR. That should help keep this component small and reduce the amount of work in this `useEffect`

Reply:
!